packages/manager/apps/dedicated/client/app/account/contacts/service/translations/Messages_de_DE.json

```diff
@@ -37,6 +37,5 @@
   "account_contacts_service_contact_billing": "Rechnungskontakt",
   "account_contacts_service_infos": "Sie können die Administator-, technischen und Rechnungs-Accounts Ihrer Dienste ändern.",
   "account_contacts_service_category_OVH_CLOUD_CONNECT": "OVHcloud Connect",
-  "account_contacts_service_category_ESSENTIALS": "Managed Bare Metal",
-  "account_contacts_service_category_LOGS": "Logs Data Platform "
+  "account_contacts_service_category_ESSENTIALS": "Essentials"
 }
```

Comment:
why key "account_contacts_service_category_LOGS" is removed? its required?

Reply:
I think this got removed while rebasing, ill add it

Reply:
i added it while fixing a dtrun issue. it neds to be added back.